source/website/src/views/Step3.vue

```diff
@@ -341,6 +341,7 @@ SPDX-License-Identifier: Apache-2.0
           { value: 'DIMENSION', text: 'Dimension', disabled: false},
           { value: 'METRIC', text: 'Metric', disabled: false},
           { value: 'isMainEventTime', text: 'MainEventTime', disabled: false},
+          { value: 'LiveRamp ID', text: 'ramp_id', disabled: false},
```

Comment:
I recommend using `text: 'LiveRamp ID'` instead of `text: 'ramp_id'`, since that is the text users will actually see.